.infrastructure/helm-chart/todoapp/templates/configMap.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{.Chart.Name}}-config
+  namespace: {{.Values.todoapp.namespace}}
```

Comment:
Ensure that the `todoapp.namespace` value is defined in your `values.yaml` file for the todoapp chart. If it's not defined, this could lead to issues when deploying the chart. Consider providing a default value or handling the case where it might be undefined.